sass/directives/02_base_components/_modals.scss

```diff
@@ -31,3 +31,16 @@
     text-decoration: none;
   }
 }
+
+@mixin close-x-overflow {
+  width: 25px;
```

Comment:
should these values be `$feature-modal-controls-size`? (only if they're meant to be the same size rather than they just happen to be the same size)